server/src/test/java/io/spine/core/RejectionsShould.java

```diff
@@ -44,6 +45,12 @@
 import static org.junit.Assert.assertTrue;
 
 /**
+ * Tests for {@link Rejections} utility class.
+ *
+ * <p>The test suite is located under the "server" module since actor request generation
```

Comment:
Let's discuss this as well. I don't really understand, why the issue exists in the first place.

Reply:
As discussed, leaving this as is.